### DISCUSSION TITLE
Customizing Static Pages with Blocks Plugin - Removing Default Content Tab

### DISCUSSION THREAD

**Discussion by JohanWoudenberg** [wintercms/winter#1016] [Q&A] [answered]
Hello,

I'm relatively new to Winter CMS and currently using the Blocks plugin in combination with the Static Pages plugin. My objective is to make the interface more user-friendly for my clients by only displaying the tab related to the Blocks plugin and removing the default 'Content' tab that comes with Static Pages.

As I'm still familiarizing myself with the intricacies of Winter CMS, I'm seeking some guidance on how to achieve this. Could anyone assist me in figuring out a method to remove or hide the default 'Content' tab, so that only the Blocks tab is visible in the Static Pages plugin?

Any advice, suggestions, or tips for a Winter CMS newcomer would be greatly appreciated.

Thank you!

**Comment by bennothommo** [comment, marked as answer]
@JohanWoudenberg you sure can - when you add the Static Page component to your page or layout, there's a setting called "Use page content field" which is ticked by default. You may untick this and it will remove the "Content" editor and tab.

**Comment by AIC-BV** [comment]
You can also add this to your plugin's boot method. This automatically removes the content field for all static pages and not a page individually.
```
// remove default content field
Event::listen('backend.form.extendFields', function($widget) {
    
    // Only apply this listener when the Page controller is being used
    if (!$widget->getController() instanceof \Winter\Pages\Controllers\Index) {
        return;
    }

    // Only apply this listener when the Page model is being modified
    if (!$widget->model instanceof \Winter\Pages\Classes\Page) {
        return;
    }

    // Only apply this listener when the Form widget in question is a root-level
    // Form widget (not a repeater, nestedform, etc)
    if ($widget->isNested) {
        return;
    }

    // remove content field
    $widget->removeField('markup');

});
```

**Comment by JohanWoudenberg** [comment]
Thank you so much for your prompt response! @bennothommo, this is a classic case of overthinking when the solution was quite straightforward. @AIC-BV , thank you for your additional input, it will definitely come in handy in another project!
